packages/bot-web-ui/src/utils/mock/ws-mock.js

```diff
@@ -0,0 +1,14 @@
+export const mock_ws = {
```

Comment:
@shafin-deriv I see you are interested to create mocking tools for WebSocket, Maybe we should discuss it 😈

Reply:
@farzin-deriv we had to pass the ws instance coming from the core to the stores. I didn't find any place to get the instance of ws so created a separate mock for it to use in all the test cases. Let's have a discussion to do it in the right way